client/src/routes/account/components/programForm/form.js

```diff
@@ -223,7 +223,7 @@ class ProgramForm extends React.Component {
               <FormGroup row>
                 <Col md={8}>
                   <Label htmlFor="category">Program Focus Area</Label>
-                  <FieldRadioBtnList name="category"
+                  <FieldCheckboxList name="category"
```

Comment:
should this not be FieldRadioList instead? different behaviour

Reply:
oh ok .. get back to this ..

Reply:
@elisechant .. didnt realise these are radio buttons ...